src/i18n_data.ts

```diff
@@ -50,24 +50,327 @@ export const resources = {
+        extensions: 'Allowed file formats: .jpg, .png, .pdf, .docx, .pptx, .zip',
+        maxSize: 'Max file size: 10 MB',
+        autoDeleteAlert: 'The file will be deleted 30 days after the end of the consultation, and will not be available afterwards!',
+        deleteKonzi: 'Delete consultation',
+        confirmDeleteKonzi: 'Are you sure you want to delete this consultation?',
+        targetGroups_one: 'Target group',
+        tergetGroups_other: 'Target groups'
+      },
+      editKonziPage: {
+        createdSuccess: 'Consultation successfully created!',
+        updatedSuccess: 'Consultation successfully updated!',
+        noAccess: 'Access denied',
+        authorEditOnly: "The consultation can only be edited by it's owner",
+        newKonzi: 'New consultation',
+        editKonzi: 'Edit {{title}}',
+        untitledKonzi: 'Unnamed consultation',
+        createNewKonzi: 'Create new consultation',
+        konziTitle: 'Consultation name',
+        nameEmpty: 'Name cannot be empty!',
+        nameTooLong: 'The name is too long!',
+        namePlaceholder: 'Analysis exam preperation',
+        location: 'Location',
+        locationEmpty: 'Location name cannot be empty!',
+        locationTooLong: 'Location name too long!',
+        language: 'Language',
+        hungarian: 'Hungarian',
+        english: 'English',
+        fullfillRequest: 'Fulfill request',
+        requestDesc: `If you've fulfilled the consultation request of someone else, tick the box above. Users who endorsed this request will
+        recieve a notification. Picking a request for the consultation will also determine it's subject.`, //
+        desc: 'Description',
+        back: 'Back',
+        create: 'Create',
+        save: 'Save'
+      },
+      selectors: {
+        request: 'Request consultation',
+        subject: 'Subject',
+        presenters: 'Tutors',
+        groups: 'Groups',
+        groupDesc: `If you provide one or more groups, only people who are part of the selected groups can view this consultation`,
+        addGroup: 'Célcsoport hozzáadása',
+        you: 'You',
+        delete: 'Delete',
+        atLeastOnePresenter: 'There must be at least one tutor',
+        addPresenter: 'Add tutor',
+        noneSelectedRequest: 'No request selected',
+        noneSelectedSubject: 'No subject selected',
+        subjectSelector: 'Select subject',
+        requestSelector: 'Select consultation request',
+        searching: 'Search...',
+        noResult: 'No results found'
+      },
+      markdown: {
+        edit: 'Edit',
+        preview: 'Preview',
+        help: 'Markdown help.',
+        placeholder: 'Type your markdown text here...',
+        noEmpty: 'Text cannot be empty!',
+        tooLong: 'Text too long!'
+      },
+      konziDateForm: {
+        date: 'Date',
+        start: 'Start',
+        end: 'End',
+        noPastStart: 'Start cannot be older than the present',
```

Comment:
Start time cannot be in the past - szerintem jobban hangzik

Reply:
older helyett meg szerintem a later jobban hangzik, meg a present is kicsit fura nekem magában, esetleg present time?

Vagy akkor már írhatnánk máshogy az egészet pl
You cannot start a consultation in the past
Th consultation cannot start in the past

Reply:
Írom a "The consultation cannot start in the past"-ot, szerintem az illik